hazelcast/src/main/java/com/hazelcast/config/NearCacheConfigReadOnly.java

```diff
@@ -81,9 +81,4 @@ public NearCacheConfig setLocalUpdatePolicy(LocalUpdatePolicy localUpdatePolicy)
     public NearCacheConfig setEvictionConfig(EvictionConfig evictionConfig) {
         throw new UnsupportedOperationException("This config is read-only");
     }
-
-    @Override
-    public EvictionConfig getEvictionConfig() {
-        return super.getEvictionConfig().getAsReadOnly();
```

Comment:
This was causing a call of `checkEvictionConfig()` which doesn't work for some Near Cache configurations. Need to fix this separately. But this causes some EE tests to fail at the moment.